prime-router/src/main/kotlin/fhirengine/azure/FHIRFunctions.kt

```diff
@@ -65,12 +65,13 @@ class FHIRFunctions(
         val messagesToDispatch = runFhirEngine(message, dequeueCount, fhirEngine, actionHistory)
         messagesToDispatch.forEach {
             queueAccess.sendMessage(
-                elrRoutingQueueName,
+                elrDestinationFilterQueueName,
                 it.serialize()
             )
         }
     }
 
+    // TODO: deprecated (see https://github.com/CDCgov/prime-reportstream/issues/15039)
```

Comment:
Is there a reason you're deprecating with TODOs and not with the Deprecated annotation?

Reply:
Historically that's how we've handled code that is pending removal/cleanup that isn't externally consumed. I reworded the comment to clean-up for now and updated the referencing ticket

I'm not 100% I should deprecate with an annotation but happy to do so if we feel strongly.